components/alibi-detect-server/adserver/protocols/__init__.py

```diff
@@ -5,6 +5,7 @@ class Protocol(Enum):
     tensorflow_http = "tensorflow.http"
     seldon_http = "seldon.http"
     seldonfeedback_http = "seldonfeedback.http"
+    v2_http = "v2.http"
```

Comment:
for consistency do you want to call it v2.http instead of kfserving? Conscious that this is kfserving in executor

Reply:
done